docs/source/essentials/server.md

```diff
@@ -203,4 +203,4 @@ server.listen({ port: config.port }, () =>
 
```

Comment:
Should this `essentials/server.md` file be deleted entirely?  It seems to preserve a fair bit of the content that was moved to `getting-started.md` and I can't tell if it was intentional or not.  It doesn't seem to be linked to from the sidebar anymore?